src/detection/detection-card/title/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { CardHeader } from '@material-ui/core'
+
+import { useStyles } from 'detection/detection-card/title/style'
+
+export const Title = () => <CardHeader title={'Detection name'}
```

Comment:
Isn't there a way of providing these props so that they all fit on the same line?